assets/js/blocks/checkout/textarea/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { TextareaControl } from '@wordpress/components';
+
+registerBlockType( 'woocommerce/checkout-textarea', {
+	title: __( 'Checkout textarea', 'woo-gutenberg-products-block' ),
```

Comment:
I think we can capitalize the second word here to match other blocks.
```suggestion
	title: __( 'Checkout Textarea', 'woo-gutenberg-products-block' ),
```